design-system/packages/fields/src/DateInput.tsx

```diff
@@ -0,0 +1,83 @@
+/** @jsx jsx */
```

Comment:
I don't think this is used right now? If not, could we remove it until we need it?

Reply:
`css` prop is being used by the DateInput in this file, though whether or not we need the DateInput component now is a good question. Let me validate tmr

Reply:
Yeah, I was referring to the DateInput, not the pragma

Reply:
- DateInput file deleted